### PR TITLE
HBASE-22908 Link To HBase 1.x Documentation on HBase Site

### PR DESCRIPTION
**Pull request by bea0113**
Added links to HBase 1.4 Ref guide and API.

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -119,6 +119,10 @@
       <item name="Bulk Loads" href="book.html#arch.bulk.load" target="_blank" />
       <item name="Metrics" href="metrics.html" target="_blank" />
       <item name="Cluster replication" href="book.html#replication" target="_blank" />
+      <item name="1.4 Documentation">
+        <item name="API" href="1.4/apidocs/index.html" target="_blank" />
+        <item name="Ref Guide (single-page)" href="1.4g/book.html" target="_blank" />
```
Comment (busbey) [inline src/site/site.xml]: the "g" here is a typo.

Comment (bea0113) [reply src/site/site.xml]: thanks, I fixed it

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  8s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 32s |  master passed  |
| +1 :green_heart: |  mvnsite  |  18m  4s |  master passed  |
| -0 :warning: |  patch  |  23m 51s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   4m 59s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |  17m 57s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 21s |  The patch does not generate ASF License warnings.  |
|  |   |  49m 17s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-993/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/993 |
| Optional Tests | dupname asflicense mvnsite xml |
| uname | Linux 89721b4c96bd 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-993/out/precommit/personality/provided.sh |
| git revision | master / 33f45d4413 |
| Max. process+thread count | 96 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-993/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  6s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 26s |  master passed  |
| +1 :green_heart: |  mvnsite  |  17m 58s |  master passed  |
| -0 :warning: |  patch  |  23m 39s |  Used diff version of patch file. Binary files and potentially other changes not applied. Please rebase and squash commits if necessary.  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m  1s |  the patch passed  |
| +1 :green_heart: |  mvnsite  |  18m  2s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 24s |  The patch does not generate ASF License warnings.  |
|  |   |  49m 16s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-993/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/993 |
| Optional Tests | dupname asflicense mvnsite xml |
| uname | Linux a01279e01923 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-993/out/precommit/personality/provided.sh |
| git revision | master / 33f45d4413 |
| Max. process+thread count | 96 (vs. ulimit of 10000) |
| modules | C: . U: . |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-993/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**